erpnext/selling/page/point_of_sale/pos_payment.js

```diff
@@ -584,7 +584,7 @@ erpnext.PointOfSale.Payment = class {
 		const remaining = grand_total - doc.paid_amount;
 		const change = doc.change_amount || remaining <= 0 ? -1 * remaining : undefined;
 		const currency = doc.currency;
-		const label = change ? __("Change") : __("To Be Paid");
+		const label = change ? __("Change Amount") : __("Change Amount");
```

Comment:
Both the true and false branches of the ternary operator are the same.

Reply:
Yes but logically thinking from sales person view it is right

Reply:
> Yes but logically thinking from sales person view it is right

That's okay, I was talking about the way you fix it. It should be `const label = __("Change Amount");` not `const label = change ? __("Change Amount") : __("Change Amount");`

Reply:
Will do new PR

Reply:
Make the change in this PR only?